### ISSUE TITLE
When stepping into the table the title of the table and number of rows and columns should be read by NVDA screenreader

### ISSUE THREAD

**Issue by mlb76** [qlik-oss/sn-table#165]
When stepping into the table the title of the table and number of rows and columns should be read by NVDA screenreader.
